modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.server/src/main/scala/edu/gemini/seqexec/web/server/http4s/SeqexecUIApiRoutes.scala

```diff
@@ -97,14 +97,13 @@ class SeqexecUIApiRoutes(auth: AuthenticationService, q: engine.EventQueue, se:
         val user = userInRequest(req)
         user.fold(Unauthorized(Challenge("jwt", "seqexec"))) { _ =>
           val r = for {
-            obsId <- \/.fromTryCatchNonFatal(new SPObservationID(oid)).leftMap((t:Throwable) => Unexpected(t.getMessage))
-            s     <- se.odbProxy.read(obsId)
+            obsId <- EitherT[Task, SeqexecFailure, SPObservationID](Task.delay(\/.fromTryCatchNonFatal(new SPObservationID(oid)).leftMap((t:Throwable) => Unexpected(t.getMessage))))
```

Comment:
In this case the `EitherT` is shortlived so I don't think it will create any problems now, but if we ever we need to combine it with another monad transformer it can get hairy.

For now I think this is fine.